episodes/severity-static.Rmd

```diff
@@ -0,0 +1,640 @@
+---
+title: 'Estimate severity'
+teaching: 10
+exercises: 2
+editor_options: 
+  chunk_output_type: inline
+---
+
+:::::::::::::::::::::::::::::::::::::: questions 
+
+- How to estimate the Case Fatality Ratio (CFR)?
+
+- Why adjust a CFR with a delay distribution?
+
+- When should we estimate the CFR in an ongoing outbreak?
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: objectives
+
+- Estimate the CFR from aggregated case data using `{cfr}`.
+
+- Estimate a delay-adjusted CFR using `{epiparameter}` and `{cfr}`.
+
+- Estimate severity measurements like IFR or HFR using `{cfr}`.
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: prereq
+
+## Prerequisites
+
+This episode requires you to be familiar with:
+
+**Data science** : Basic programming with R.
+
+**Epidemic theory** : Epidemiological parameters. Time periods.
+
+:::::::::::::::::::::::::::::::::
+
+## Introduction
+
+Common questions at the early stage of an epidemic include:
+
+- What is the likely public health impact of the outbreak? 
+- Is this being or going to be a public health problem?
+- Does the virus have the potential to cause a devastating pandemic?
+
+We can assess the pandemic potential of an epidemic with two critical measurements: the transmissibility and the clinical severity. ([Fraser et al., 2009](https://www.science.org/doi/full/10.1126/science.1176062), [CDC, 2016](https://www.cdc.gov/flu/pandemic-resources/national-strategy/severity-assessment-framework-508.html))
```

Comment:
Thanks for the detailed edit suggestion. I agree with the need to homogenize this to outbreaks or epidemics, we are currently using this indistinctively in the episode (and other documentation). I think this would be appropriate content for a callout #6 . I'm going to prioritize the edits for the questions.

The first one is from [Cori et al.](https://royalsocietypublishing.org/doi/10.1098/rstb.2016.0371#d1e605) and just frame it to be specific to this episode. 

We can reuse your last paragraph.

```suggestion
## Introduction

Common questions at the early stage of an epidemic include:

- What is the likely public health impact of the outbreak in terms of clinical severity? 
- What are the most severely affected groups?
- Does the outbreak have the potential to cause a very severe pandemic?

We can assess the pandemic potential of an epidemic with two critical measurements: the transmissibility and the clinical severity. ([Fraser et al., 2009](https://www.science.org/doi/full/10.1126/science.1176062), [CDC, 2016](https://www.cdc.gov/flu/pandemic-resources/national-strategy/severity-assessment-framework-508.html))
```

Reply:
paragraph added in https://github.com/epiverse-trace/tutorials-middle/pull/1/commits/e93b7560129808b7ed049425503b0201cdb8fe11